keiko/templates/bevindingenrapport/model.py

```diff
@@ -10,23 +10,23 @@
 
 
 class OOI(BaseModel):
-    id: str
-    ooi_type: str
-    human_readable: str
-    object_type: str
+    id: Optional[str]
```

Comment:
Can we just do this? How does jinja handle None's? Shouldn't we make sure these fields are set for OOIs from Rocky?

Reply:
I'm currently writing a feature request to standardize this (data shape) format